### ISSUE TITLE
user_agent opt doesn't do anything

### ISSUE THREAD

**Issue by gmkbenjamin** [WinRb/WinRM#344]
Tried setting user_agent option but it doesn't seem to do anything. Still getting the default 'Ruby WinRM Client (2.8.3, ruby 3.1.2 (2022-04-12))' useragent string.

**Comment by pcai** [comment]
The user_agent change hasn't been released on rubygems yet. Are you using winrm from master? If not, try something like this in your gemfile:

```
git_source(:github) { |repo| "https://github.com/#{repo}.git" }
gem 'winrm', github: 'WinRb/winrm'
```

I was recently added as a maintainer so I can try to cut a release next week. Please reopen if you can confirm the user-agent option isn't working on master.

**Comment by gmkbenjamin** [comment]
Hello,
I can confirm that master works. Could you cut a release for rubygems please? I have projects depend on this. Thank you. :)

**Comment by pcai** [comment]
Apologies for the delay - slipped my mind. Will get a release out today

**Comment by pcai** [comment]
released as v2.3.7
